experimental/generation/generator/templates/library-Missing.en-us.lg

```diff
@@ -1,48 +1,48 @@
 [library.en-us.lg]
 [library-Help.en-us.lg]
 
-# askEnum(property)
+# getAskForEnumText(property)
```

Comment:
>getAskForEnumText [](start = 2, length = 17)

Rather than Ask, thought we were using 'prompt'.

Reply:
Changed to 
getEnumPromptText

---
In reply to: [433372969](https://github.com/microsoft/BotBuilder-Samples/pull/2461#discussion_r433372969) [](ancestors = 433372969)